# HG changeset patch
# User Rik <rik@octave.org>
# Date 1349104985 25200
#      Mon Oct 01 08:23:05 2012 -0700
# Node ID 6a05cad5b694a9c380bb1a173befb066ee6d1ee2
# Parent  e8b117b405f93412798ba45db1706e612d345352
Fix first-pass compiling which creates bad *.df files (bug #37461)

* libinterp/Makefile.am: Re-order BUILT_SOURCES list so that all necessary
files are created before making builtin-defun-decls.h

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -48,23 +48,23 @@ octlib_LTLIBRARIES = liboctinterp.la
 ## $(DEF_FILES), and building those requires all the sources
 ## (except builtins.cc) to be available.
 BUILT_SOURCES = \
   interp-core/mxarray.h \
   interp-core/oct-errno.cc \
   interpfcn/defaults.h \
   interpfcn/graphics-props.cc \
   interpfcn/graphics.h \
-  builtin-defun-decls.h \
   operators/ops.cc \
   parse-tree/lex.cc \
   parse-tree/oct-gperf.h \
   parse-tree/oct-parse.cc \
   oct-conf.h \
   version.h \
+  builtin-defun-decls.h \
   builtins.cc
 
 BUILT_DISTFILES = \
   parse-tree/oct-gperf.h \
   parse-tree/oct-parse.h
 
 ## Files that are created during build process and installed,
 ## BUT not distributed in tarball.
